social/tasks.py

```diff
@@ -0,0 +1,44 @@
+import feedparser
+import re
+
+from background_task import background
+
+from social.app.models.author import Author
+from social.app.models.post import Post
+
+# Get the GitHub activity of a user
+# Reference source: https://pypi.python.org/pypi/django-background-tasks
+@background(schedule=60)
+def get_github_activity(authorId):
+    gitAuthor = Author.objects.get(id=authorId)
+    gitUrl = gitAuthor.github
+
+    # Using RegEx to check if it's a proper URL
+    # Reference source: https://github.com/lorey/social-media-profiles-regexs#github
+    # TODO: Fix this so that it won't continue if given something like /user/repo
+    if(r'http(s)?:\/\/(www\.)?github\.com/[A-z 0-9 _ -]+\/?$'):
```

Comment:
This just returns true always, because all you have here is a string, and a non-empty string has a true-y value. You're not actually using the `re` module to test against anything. You need to use something like this: https://docs.python.org/2/library/re.html#re.match